src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-handled-exceptions.mdx

```diff
@@ -0,0 +1,387 @@
+---
+title: Record handled exception
+type: apiDoc
+shortDescription: Records a handled exception. Optionally takes map with additional attributes showing context.
+tags:
+  - Mobile monitoring
+  - Mobile SDK API
+  - Custom instrumentation
+metaDescription: Mobile app monitoring API used to analyze handled exceptions and thrown exceptions.
+redirects:
+  - /docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/recordhandledexception-ios-sdk-api
+  - /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/recordhandledexception
+  - /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/recordhandledexception-android-sdk-api
+  - /docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/record-handled-exception
+  - /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/record-jserror-exception/
+---
+
+<Tabs>
+	<TabsBar>
+        <TabsBarItem id="android">
+            Android
+        </TabsBarItem>
+        <TabsBarItem id="ios">
+            iOS
+        </TabsBarItem>
+        <TabsBarItem id="xamarin">
+            Xamarin
+        </TabsBarItem>
+        <TabsBarItem id="maui">
+            .NET MAUI
+        </TabsBarItem>
+    </TabsBar>
+
+    <TabsPages>
+        <TabsPageItem id="android">
+## Syntax [#syntax]
+
+### Java [#java]
+
+```java
+NewRelic.recordHandledException(Exception $exceptionToHandle)
+NewRelic.recordHandledException(Exception $exceptionToHandle, Map of String, Object $exceptionAttributes)
+```
+
+```java
+NewRelic.recordHandledException(Throwable $throwableToHandle)
+NewRelic.recordHandledException(Throwable $throwableToHandle, Map of String, Object $exceptionAttributes)
+```
+
+### Kotlin [#kotlin]
+
+```kotlin
+  NewRelic.recordHandledException(exception: Exception?)
+```
+
+```kotlin
+
+```
```

Comment:
@ndesai-newrelic Can you please provide a kotlin example equivalent to the Java example?

Reply:
please add this in syntax: NewRelic.recordHandledException(
            exception: Exception?,
            exceptionAttributes: Map<String?, Any?>?
        )

2) NewRelic.recordHandledException(throwable: Throwable?)
NewRelic. recordHandledException(
            throwable: Throwable?,
            attributes: Map<String?, Any?>?
        )